docs/src/pages/components/data-grid/editing/ValidateCellApiRefGrid.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  GridColumns,
+  GridEditCellPropsParams,
+  GridRowsProp,
+  useGridApiRef,
+  XGrid,
+} from '@material-ui/x-grid';
+import {
+  randomCreatedDate,
+  randomEmail,
+  randomTraderName,
+  randomUpdatedDate,
+} from '@material-ui/x-grid-data-generator';
+
+const columns: GridColumns = [
+  { field: 'name', headerName: 'Name', width: 180, editable: true },
+  { field: 'email', headerName: 'Email', width: 200, editable: true },
+  {
+    field: 'dateCreated',
+    headerName: 'Date Created',
+    type: 'date',
+    width: 180,
+    editable: true,
+  },
+  {
+    field: 'lastLogin',
+    headerName: 'Last Login',
+    type: 'dateTime',
+    width: 220,
+    editable: true,
+  },
```

Comment:
Since the only validation is on the email, it seems a lot better to remove all the other columns, to guide the developers to only what they need to pay attention to:

```suggestion
```